ntcore/src/main/native/cpp/LocalStorage.cpp

```diff
@@ -54,7 +54,7 @@ int LocalStorage::DataLoggerData::Start(TopicData* topic, int64_t time) {
     typeStr = "int64[]";
   }
   return log.Start(fmt::format("{}{}", logPrefix,
-                               wpi::drop_front(topic->name, prefix.size())),
+                               wpi::remove_prefix(topic->name, prefix).value()),
```

Comment:
We really don't want this throwing an exception.  I think we can just fall back on topic->name.